baremaps-cli/src/main/java/com/baremaps/cli/Edit.java

```diff
@@ -111,4 +120,26 @@ protected void configure() {
 
     return 0;
   }
+
+  @Provider
+  private static class MyObjectMapperProvider implements ContextResolver<ObjectMapper> {
```

Comment:
This is a duplicate right? I would make this class public in the baremaps-openapi module to reuse it here.

Reply:
Yes, i tried once again to address #320 but ended up with this. Sorry!

Reply:
No problem ;-) Let's rename this class and make public. What about `ObjectMapperProvider`?

Reply:
Done in https://github.com/baremaps/baremaps/pull/338/commits/0990c395e436b9fb590168b906702d2986b061e5